src/Options.php

```diff
@@ -636,8 +659,9 @@ private function configureOptions(OptionsResolver $resolver): void
         $resolver->setAllowedTypes('error_types', ['int']);
         $resolver->setAllowedTypes('max_breadcrumbs', 'int');
         $resolver->setAllowedTypes('before_breadcrumb', ['callable']);
-        $resolver->setAllowedTypes('integrations', ['null', 'array']);
+        $resolver->setAllowedTypes('integrations', 'array');
         $resolver->setAllowedTypes('send_default_pii', 'bool');
+        $resolver->setAllowedTypes('default_integrations', 'bool');
```

Comment:
I think that the name of this option cames from the unified API so this suggestion is not related to this particular change/package: wouldn't be better and more clear to call it `use_default_integrations`?